minecraft_servers/skycraftbr/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "server_name": "skycraftbr",
+    "nice_name": "SkyCraft Network",
+    "direct_ip": "original.skycraft.com.br",
+    "server_wildcards": [
+      "%.skycraft.com.br",
+      "%.skycraft.network",
+      "%.skycraft.games"
+    ],
+    "supported_languages": [
+       "pt"
+    ],
+    "social": {
+      "web": "https://skycraft.com.br",
+      "web_shop": "https://loja.skycraft.com.br",
```

Comment:
Can't reach website

Reply:
Both websites are offline temporarely, will be up soon.

Reply:
How long is "soon"?

Reply:
This week. We've rebuilt the server from scratch, so it'll take just a little time to set up the shop and site. It's a matter of days

Reply:
I'll keep the PR open until then.
Just write a message in this PR once they're online

Reply:
Isn't the website optional?

Reply:
It is optional. Either you remove the keys until they're online or the PR stays open until they're online.

Reply:
Done.